pkg/verifier/aws/aws_verifier.go

```diff
@@ -521,16 +522,35 @@ func (a *AwsVerifier) CreateSecurityGroup(ctx context.Context, tags map[string]s
 // ipPermissionFromURL generates an EC2 IpPermission (for use in sec. group rules) from a given IP address
 // -based http(s) URL (e.g. "https://10.0.8.1:8080") with the given human-readable description (ipPermDescription)
 func ipPermissionFromURL(ipUrlStr string, ipPermDescription string) (*ec2Types.IpPermission, error) {
-	// First validate URL by parsing it
+
+	//validate URL by parsing it
 	ipUrl, err := url.Parse(ipUrlStr)
 	if err != nil {
 		return nil, err
 	}
 
-	// Then attempt to extract IP address
-	ipAddr, err := netip.ParseAddr(ipUrl.Hostname())
+	//extract the hostName from the URL
+	hostValue := ipUrl.Hostname()
+	ipFromDomain := hostValue
+
+	//start a new validator to help verify the hostname
+	validate := validator.New()
+
+	//check if the hostname is a fully qualified domain name
+	err = validate.Var(hostValue, "fqdn")
+	if err != nil {
+		//do nothing in order to proceed to check if hostValue is an IP address
+	} else {
+		//if the hostValue is a fqdm set the ip to 0.0.0.0 in order to create an Outbound SG rule to all IPs
+		//Ref: OSD-20562
+		ipFromDomain = "0.0.0.0"
+	}
+
+	////Check if the hostname is an IP address
```

Comment:
Extra comment lines can be removed here
```suggestion
	// Check if the hostname is an IP address
```